lib/elastomer/client/delete_by_query.rb

```diff
@@ -48,29 +48,32 @@ def delete_by_query(query, params = {})
       DeleteByQuery.new(self, query, params).execute
     end
 
-    SEARCH_PARAMETERS = [
-      :q,
-      :df,
-      :analyzer,
-      :analyze_wildcard,
-      :batched_reduce_size,
-      :default_operator,
-      :lenient,
-      :explain,
-      :_source,
-      :stored_fields,
-      :sort,
-      :track_scores,
-      :timeout,
-      :terminate_after,
-      :from,
-      :size,
-      :search_type,
-      :scroll
-    ].to_set.freeze
-
     class DeleteByQuery
 
+      SEARCH_PARAMETERS = %i[
+        index
+        type
```

Comment:
To be clear, these were the two parameters that were missing, right?

Reply:
Correct. These were being stripped out of the params passed to the scroll query.